src/app/components/MostRead/index.tsx

```diff
@@ -84,6 +88,14 @@ const MostRead = ({
       </MostReadSection>
     ) : null;
 
+  const eventTrackingData = {
+    ...blockLevelEventTrackingData,
+    ...(sendOptimizelyEvents && {
+      optimizely,
+      optimizelyMetricNameOverride: 'most_read',
+    }),
```

Comment:
We need these checks because the JumpTo component doesn't always show in general. It will only show for articles with 3-8 subheadings, so we don't want to fire Optimizely events when that condition isn't met.